sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -179,13 +179,7 @@ export class KeyVaultAccessControlClient {
     options: DeleteRoleAssignmentOptions = {}
   ): Promise<void> {
     return withTrace("deleteRoleAssignment", options, async (updatedOptions) => {
-      try {
-        await this.client.roleAssignments.delete(this.vaultUrl, roleScope, name, updatedOptions);
-      } catch (error) {
-        if (error.statusCode !== 404) {
-          throw error;
-        }
-      }
+      await this.client.roleAssignments.delete(this.vaultUrl, roleScope, name, updatedOptions);
```

Comment:
You shouldn't return a raw response or anything?

Reply:
Yes, I can return the raw response. Thanks!

Reply:
Only if that's what you'd usually do. When I opened the issue, I probably should've said "return no value" instead of "return void".

Reply:
it's a great callout. Unfortunately there is some inconsistency between returning `Promise<void>` and `Promise<RestResponse>` for these operations. I went with `Promise<void>` because it seemed more common at first but talking to our architects we agree returning a raw response is good here.